packages/utils/src/fileToGatewayUrl.ts

```diff
@@ -13,7 +14,10 @@ export function fileToGatewayUrl(distributedFile?: DistributedFile): string {
 
   switch (distributedFile.source) {
     case "ipfs":
-      return `${params.IPFS_GATEWAY}${normalizeHash(distributedFile.hash)}`;
+      return url.resolve(
```

Comment:
What about:
```
return url.resolve(
  `${params.IPFS_REMOTE}/${params.IPFS_GATEWAY}`,
  normalizeHash(distributedFile.hash)
);
```
to avoid using `url.resolve` twice here?